types/src/core/strategies/transport_strategy.d.ts

```diff
@@ -9,6 +9,9 @@ export default class TransportStrategy implements Strategy {
     constructor(name: string, priority: number, transport: Transport, options: StrategyOptions);
     isSupported(): boolean;
     connect(minPriority: number, callback: Function): {
+        abort: () => void;
```

Comment:
Why is this needed? 💭

Reply:
I didn't add this. I think it was added automatically after I ran the commands to format the code and/or running the unit and integration tests.